CMakeLists.txt

```diff
@@ -2,53 +2,133 @@
 # Check version meets our requirements
 # Declare project, which will configure compiler for us
 cmake_minimum_required(VERSION 3.3)
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
Like the rest of these lines in the commits, these are conflicts and should have been resolved. See tutorials on how to resolve these: https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts

As it stands, the module is completely unbuildable.